*: improve some links (#184)
pingcap/community: README.md

![TiDB Community Architecture](./media/architecture.svg)

> See [`architecture` directory](./architecture/README.md) for details.

Reviewer: ```suggestion
See [`architecture`](./architecture/README.md) for details.
```

## TiDB User Group

The TiDB User Groups (TUGs) are groups for facilitating communication and
discovery of information related to topics that have long term relevance to
large groups of TiDB users.

See [pingcap/docs](https://pingcap.com/docs/) or
[pingcap/docs-cn](https://pingcap.com/docs-cn/) for user documents. Also you can get
help in [AskTUG.com](https://asktug.com/) if you met any problem.

## TiDB Developer Group

### Communication

The [communicating.md](./communicating.md) file lists communication channels like
chat, social medias, etc.

For more specific topics, join

> [TiDB Community slack channel](https://join.slack.com/t/tidbcommunity/shared_invite/enQtNzc0MzI4ODExMDc4LWYwYmIzMjZkYzJiNDUxMmZlN2FiMGJkZjAyMzQ5NGU0NGY0NzI3NTYwMjAyNGQ1N2I2ZjAxNzc1OGUwYWM0NzE) and discuss with others.

Reviewer: ```suggestion
[TiDB Community slack workspace](https://join.slack.com/t/tidbcommunity/shared_invite/enQtNzc0MzI4ODExMDc4LWYwYmIzMjZkYzJiNDUxMmZlN2FiMGJkZjAyMzQ5NGU0NGY0NzI3NTYwMjAyNGQ1N2I2ZjAxNzc1OGUwYWM0NzE) and discuss with others.
```

  module of TiDB. SIGs must have open and transparent proceedings. The purpose
  of a SIG is to own and develop a module of TiDB.

>   See [`special-interest-groups` directory](./special-interest-groups) for details and active SIGs.

Reviewer: ```suggestion
  See [`special-interest-groups`](./special-interest-groups/README.md) for details and active SIGs.
```

>   See [`special-interest-groups` directory](./special-interest-groups/README.md) for details and active SIGs.

Reviewer: ```suggestion
  See [`special-interest-groups`](./special-interest-groups/README.md) for details and active SIGs.
```

* **Working Groups (WGs)** are temporary groups that are formed to address
  issues that cross SIG boundaries. Working groups do not own any code or other
  long term artifacts. Working groups can report back and act through involved
  SIGs.

>   See [`working-groups` directory](./working-groups) for details and active WGs.

Reviewer: ```suggestion
  See [`working-groups`](./working-groups/README.md) for details and active WGs.
```

### How to contribute

Contributions are welcomed and greatly appreciated.

> See [`contributors` directory](./contributors/README.md) for details.

Reviewer: ```suggestion
See [`contributors`](./contributors/README.md) for details.
```

## Learning Resources

> Learning resources are collected in the [`learning-resources` directory](./learning-resources/README.md). Here you can find all the

Reviewer: ```suggestion
Learning resources are collected in the [`learning-resources`](./learning-resources/README.md). Here you can find all the
```

## RFC

> See [`rfc` directory](./rfc/README.md).

Reviewer: ```suggestion
See [`rfc`](./rfc/README.md).
```
